src/components/countries/Countries.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import  { COUNTRIES__QUERY } from "../../graphql/queries/countries";
+import { WithQueryCountries } from "../hoc/withQueryCountries";
+
+import { Country } from "./country/Country";
+import "./countries.scss";
+
+export const Countries = WithQueryCountries(({ data }) => {
+  return (
+    <div className="countries">
+      <>{!data && <div> The data about countries is not exist</div>}</>
+      <div className="countries__choose-card choose-card">
+        <div className="choose-card__img">
+          <img src="/smile2.png" alt="smile" />

```

Comment:
Please move to separate component and use SVG.
```suggestion
import SelectCardIcon from './SelectCardIcon';
     ...
         <div className="select-card__img" dangerouslySetInnerHTML={{__html: SelectCardIcon}} />
```

```suggestion
const SelectCardIcon = `<svg ... </svg>;
export default SelectCardIcon;
```